crates/mun_runtime/src/lib.rs

```diff
@@ -160,14 +148,22 @@ mod tests {
 
     #[test]
     fn mun_new_runtime() {
-        let _runtime = MunRuntime::new(&test_lib_path(), Duration::from_millis(10))
-            .expect("Failed to initialize Mun runtime.");
+        let options = RuntimeOptions {
+            library_path: test_lib_path(),
+            delay: Duration::from_millis(10),
+        };
+
+        let _runtime = MunRuntime::new(options).expect("Failed to initialize Mun runtime.");
     }
 
     #[test]
     fn mun_invoke_fn() {
-        let mut runtime = MunRuntime::new(&test_lib_path(), Duration::from_millis(10))
-            .expect("Failed to initialize Mun runtime.");
+        let options = RuntimeOptions {
```

Comment:
I prefer using a `Builder` construct instead of using options. An other option would be to implement `Default` for `RuntimeOption`